tests/unit/test_launchers.py

```diff
@@ -63,11 +76,15 @@ def test_stop_fail(self):
 
 
 class TestMozRunnerLauncher(unittest.TestCase):
+    def clean_launcher(self):
+        if hasattr(self, 'launcher'):
+            del self.launcher
```

Comment:
Totally agree. but it is here for quite a long time - good news is that it is the only last thing in mozreg that rely on destructors. A bug to fix that would be great, I'll create that.